public/js/jquery.t.js

```diff
@@ -348,6 +366,8 @@
             // Ensure clicking on 'rect' takes user to the relevant hit on all
             // browsers.
             setupClick($graphDiv);
+            // Redraw the graph on a browser resize...
+            setupResponsiveness($queryDiv, $graphDiv, opts);
```

Comment:
Are you sure this doesn't attach the same handler several times to `window.resize`?

@vivekiitkgp Can you verify the case for `setupClick` as well?